### PR TITLE
[hack] some scripts we can use for testing

### PR DESCRIPTION
**Pull request by jmazzitelli**
1. hack/istio/install-travel-agency-demo.sh
* Another microservice demo with added benefit of a couple defined operations in telemetry.
2. hack/jwt-[en,de]code.sh
* Two scripts that let us test passing in and examining JWT tokens. Useful to make sure hacking JWT tokens is not possible. Also useful to pass in when testing Kiali APIs as authenticated users.

I used these hack scripts for several things. First to confirm the correctness of our new OpenID integration. Also the demo let's us test the graph as well as producing telemetry with classified metric data (the operation ID stuff).

### CODE REVIEW

**Comment by jmazzitelli** [comment]
putting @lucasponce and/or @israel-hdez as reviewers - they are familiar with the travel agency demo and the jwt scripts respectively.

File: hack/istio/install-travel-agency-demo.sh [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+kind: NetworkAttachmentDefinition
+metadata:
+  name: istio-cni
+EOF
+fi
+
+# Deploy the demo
+
+${CLIENT_EXE} apply -f <(curl -L https://raw.githubusercontent.com/lucasponce/travel-comparison-demo/master/travel_agency.yaml) -n ${NAMESPACE_AGENCY}
+${CLIENT_EXE} apply -f <(curl -L https://raw.githubusercontent.com/lucasponce/travel-comparison-demo/master/travel_portal.yaml) -n ${NAMESPACE_PORTAL}
```
Comment (lucasponce) [inline hack/istio/install-travel-agency-demo.sh:85]: This is nice.
I wonder if we could add some parameter to also install the version 2 in an optional manner (Step [2.5.0]):
https://lucasponce.github.io/presentations/observability-in-action.html

File: hack/istio/install-travel-agency-demo.sh [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
++                     }
++                   }]
+                 }
+               root_id: stats_inbound
+               vm_config:
+EOF
+
+cat <<EOF | ${CLIENT_EXE} -n istio-system apply -f -
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
```
Comment (lucasponce) [inline hack/istio/install-travel-agency-demo.sh:119]: Nice EnvoyFilters.

File: hack/istio/install-travel-agency-demo.sh (hunk 1 above)
Comment (jmazzitelli) [reply hack/istio/install-travel-agency-demo.sh]: added that to the script

**Pull request merged**